app/static/tests/unit/components/graphConfig/graphConfig.test.ts

```diff
@@ -133,6 +154,26 @@ describe("GraphConfig", () => {
             getData: (s: string) => {
                 if (s === "variable") return "I";
                 if (s === "srcGraphConfig") return "hidden";
+                if (s === "copyVar") return "false";
+                return null;
+            }
+        };
+        const dropPanel = wrapper.find(".graph-config-panel");
+        await dropPanel.trigger("drop", { dataTransfer });
+        expect(mockUpdateSelectedVariables.mock.calls.length).toBe(1);
+        expect(mockUpdateSelectedVariables.mock.calls[0][1]).toStrictEqual({
+            graphIndex: 0,
+            selectedVariables: ["S", "R", "I"]
+        });
+    });
```

Comment:
Can we hold down a ctrl/meta key here, to show that trying to copy a hidden variable on dragging doesn't end up with it existing in both places (hiddens and drag target)